Common/Global/GlobalSpiritModItem.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.ModLoader;
+using System;
+using System.Collections.Generic;
+using TheBereftSouls.Common.Utility;
+
+namespace TheBereftSouls.Common.Global
+{
+	public class GlobalSpiritModItem : GlobalItem
+	{
+		//Untested - JIT used to prevent crashes if Calamity isn't enabled.
+		
+
+		// Changes following Spirit Weapons To Rogue
+		private static List<string> SpiritWeaponsToRogue = new List<string>
+		{
+			"BismiteGrenade",
+			"DodgeBall",
+			"DuskfeatherDagger",
+			"ExplosiveRum",
+			"Kunai_Throwing",
+			"SoaringScapula"
+		};
+		[JITWhenModsEnabled("CalamityMod","SpiritMod")]
+		public override void SetDefaults(Item item)
+		{
+			// Checks to see if Calamity and Spirit are installed before making changes.
+			if (!ModLoader.TryGetMod("CalamityMod", out Mod CalamityMod) | !ModLoader.TryGetMod("SpiritMod", out Mod SpiritMod))
+			{
+				return;
+			}
+			// Uses list to change all listed weapons to Rogue
+			foreach (string spiritWeaponName in SpiritWeaponsToRogue)
+			{
+				if (item.type == SpiritMod.Find<ModItem>(spiritWeaponName).Type)
+				{
+					item.DamageType = GetModClass.rogueDamageClass;
+					base.SetDefaults(item);
```

Comment:
Same as [this comment](https://github.com/bromeex/TheBereftSouls/pull/9/files#r1793423317 "#9 Common/Global/GlobalVanillaItem.cs Line 39 commit: eef2c06d9cbb21c3fb344de9393cc472519491b0 (\"Test Commit
\")").